python/mxnet/kvstore.py

```diff
@@ -63,6 +63,19 @@ def __init__(self, handle):
     def __del__(self):
         check_call(_LIB.MXKVStoreFree(self.handle))
 
+    def get_type(self):
```

Comment:
change this to 
```python
@property
def type(self):
```
This allow us to do ```kv.type``` directly. Although type is a reserved function in python. I guess this is fine